django/db/models/lookups.py

```diff
@@ -111,7 +111,7 @@ def get_group_by_cols(self):
             cols.extend(self.rhs.get_group_by_cols())
         return cols
 
-    def as_sql(self, compiler, connection):
+    def as_sql(self, compiler, connection, **kwargs):
```

Comment:
Agreed. Let's ditch `**kwargs` in lookups if they aren't being used for anything.